src/reference/01-General-Info/05-Using-Sonatype.md

```diff
@@ -117,8 +117,7 @@ addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.1.1")
 
 > *Note:* The plugin is a solution to sign artifacts. It works with the GPG command line tool.
 
-* make sure that the `gpg` command is in PATH available to the sbt,
-* add `useGpg := true` to your `build.sbt` to make the plugin `gpg`-aware
+Make sure that the `gpg` command is in PATH available to the sbt.
```

Comment:
Could you bump the version number a few lines above to

```scala
addSbtPlugin("com.jsuereth" % "sbt-pgp" % "2.0.1")
```

please?

Reply:
Yea, sure! I have already made it in my code but overlook to update manual ;)